flexmeasures_openweathermap/utils/owm.py

```diff
@@ -108,31 +108,34 @@ def save_forecasts_in_db(
                         weather_sensors,
                         max_degree_difference_for_nearest_weather_sensor,
                     )
-                    if weather_sensor not in db_forecasts.keys():
-                        db_forecasts[weather_sensor] = []
-
-                    fc_value = fc[owm_response_label]
-
-                    # the irradiance is not available in OWM -> we compute it ourselves
-                    if sensor_name == "irradiance":
-                        fc_value = compute_irradiance(
-                            location[0],
-                            location[1],
-                            fc_datetime,
-                            # OWM sends cloud cover in percent, we need a ratio
-                            fc_value / 100.0,
+                    if weather_sensor is not None:
+                        if weather_sensor not in db_forecasts.keys():
```

Comment:
Let's add a log here that we found this particular sensor.: "Found pre-configured weather sensor {weather_sensor.name} ..."